docs/Installation/install.md

```diff
@@ -10,7 +10,7 @@
 curl https://raw.githubusercontent.com/threefoldtech/jumpscaleX/master/install/install.py?$RANDOM > /tmp/install.py
 
 #if you want to use development branch (unstable) do:
-curl https://raw.githubusercontent.com/threefoldtech/jumpscaleX/master/install/install.py?$RANDOM > /tmp/install.py
+curl https://raw.githubusercontent.com/threefoldtech/jumpscaleX/development/install/install.py?$RANDOM > /tmp/install.py
```

Comment:
this is to be merged to master... so it should be installing master InstallTools

Reply:
yeah sure, It's just in the readme if anyone need to install from development.